src/App.tsx

```diff
@@ -17,35 +20,40 @@ export const App = () => (
     >
       <div className="container">
```

Comment:
Put nav to a Navigation component and create a Layout component that will serve as a wrapper for all your routes. Within this Layout, we'll include a Navigation component and a section to wrap the content. To achieve this, we can use the Outlet component from react-router-dom.

```
<Routes>
  <Route path="/" element={<Layout />}>
    .. homepage
    ...tabs
    ...
 </Routes>   
```